app/integration_test/README.md

```diff
@@ -1,211 +1,53 @@
-I/flutter (10196):   Actual: <4>
-I/flutter (10196): 
-I/flutter (10196): When the exception was thrown, this was the stack:
-I/flutter (10196): #4      main.<anonymous closure>
-(file:///Users/jonassharezone/development/projects/sharezone-app/app/integration_test/foo_test.dart:8:5)
-I/flutter (10196): #5      testWidgets.<anonymous closure>.<anonymous closure>
-(package:flutter_test/src/widget_tester.dart:144:29)
-I/flutter (10196): <asynchronous suspension>
-[Ausgelassen für weniger Text]
-I/flutter (10196): #35     Invoker._onRun.<anonymous closure>
-(package:test_api/src/backend/invoker.dart:369:7)
-I/flutter (10196): #42     Invoker._onRun (package:test_api/src/backend/invoker.dart:368:11)
-I/flutter (10196): #43     LiveTestController.run
-(package:test_api/src/backend/live_test_controller.dart:153:11)
-I/flutter (10196): (elided 31 frames from dart:async and package:stack_trace)
-I/flutter (10196): 
-I/flutter (10196): This was caught by the test expectation on the following line:
-I/flutter (10196):
-file:///Users/jonassharezone/development/projects/sharezone-app/app/integration_test/foo_test.dart line 8
-I/flutter (10196): The test description was:
-I/flutter (10196):   failing test example
-I/flutter (10196):
-════════════════════════════════════════════════════════════════════════════════════════════════════
-I/flutter (10196): 00:00 +0: failing test example [E]
-I/flutter (10196):   Test failed. See exception logs above.
-I/flutter (10196):   The test description was: failing test example
-I/flutter (10196):   
-I/flutter (10196): 00:00 +0 -1: succeeding test example
-Failure Details:
-Failure in method: failing test example
-══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞═════════════════
-The following TestFailure object was thrown running a test:
-  Expected: <5>
-  Actual: <4>
-
-When the exception was thrown, this was the stack:
-#4      main.<anonymous closure> (file:///Users/jonassharezone/development/projects/sharezone-app/app/integration_test/foo_test.dart:8:5)
-#5      testWidgets.<anonymous closure>.<anonymous closure> (package:flutter_test/src/widget_tester.dart:144:29)
-[Ausgelassen für weniger Text]
-#35     Invoker._onRun.<anonymous closure> (package:test_api/src/backend/invoker.dart:369:7)
-#42     Invoker._onRun (package:test_api/src/backend/invoker.dart:368:11)
-#43     LiveTestController.run (package:test_api/src/backend/live_test_controller.dart:153:11)
-(elided 31 frames from dart:async and package:stack_trace)
-
-This was caught by the test expectation on the following line:
-  file:///Users/jonassharezone/development/projects/sharezone-app/app/integration_test/foo_test.dart line 8
-The test description was:
-  failing test example
-═════════════════════════════════════════════════════════════════
-
-end of failure 1
-
-
-
-Stopping application instance.
-Driver tests failed: 1
+### Mobile
+You can run the integration tests using the `flutter drive` command:
+```sh
+flutter test \
```

Comment:
Shouldn't we use `fvm flutter` for the examples?

Reply:
Yes makes more sense when we use `fvm`